pkg/specgen/generate/validate.go

```diff
@@ -48,7 +48,7 @@ func verifyContainerResourcesCgroupV1(s *specgen.SpecGenerator) ([]string, error
 				warnings = append(warnings, "Your kernel does not support memory swappiness capabilities, or the cgroup is not mounted. Memory swappiness discarded.")
 				memory.Swappiness = nil
 			} else {
-				if *memory.Swappiness < 0 || *memory.Swappiness > 100 {
```

Comment:
Why is the less than zero check removed?

Reply:
The swappiness type is uint64 and therefore cannot be less than zero.

Reply:
I'd just be concerned that someone somewhere down the road decided to change the variable to a signed int.  I'd keep the check unless it causes real headaches.  Others may have differing opinions.

Reply:
`Swappiness` is defined [here](https://github.com/opencontainers/runtime-spec/blob/7413a7f753e1bd9a6a9c6dc7f96f55888cbbd476/specs-go/config.go#L308). It is part of the OCI runtime specification. I highly doubt that this will ever be changed to a signed integer.